content/java/concepts/math-methods/terms/floor/floor.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: 'Floor'
+Description: 'Returns the largest integer value that is less than or equal to the argument. When the provided value is either an integer, zero, not a number, or positive or negative infinity.'
+Subjects: 
+    -'Code Foundations'
+    -'Computer Science'
+Tags: 
+    - 'Documentation'
+CatalogContent: 
+  - 'learn-java/static-methods-of-the-math-class'
+---
+
+The static function **`.floor()`** included in the `Math` class returns the largest integer value that is less than or equal to the argument. When the provided value is either an integer, zero, not a number, or positive or negative infinity.
+
+## Syntax
+The Math package can be imported and utilized as shown below
+```java
+    import java.lang.Math;
+
+    double x = 1.1;
+    Math.floor(1.1); // passing values directly to the function
+    Math.floor(x); // or passing values of variables
+```
+
+## Example 1
```

Comment:
Let's consolidate these into two examples that fit this general structure/format:

## Example 1

[Sentence(s)]

[\```java code block] <--- Successful calls that include negative values.

## Example 2

[Sentence(s)]

[\```java code block] <-- Unsuccessful calls that include `NaN` and Infinity.

Reply:
Simplified and consolidated examples as suggested.